static/app/views/performance/trends/changedTransactions.tsx

```diff
@@ -298,7 +298,7 @@ function ChangedTransactions(props: Props) {
           <TransactionsListContainer data-test-id="changed-transactions">
             <TrendsTransactionPanel>
               <StyledHeaderTitleLegend>
-                {chartTitle}
+                {chartTitle} {!withBreakpoint && 'Indexed'}
```

Comment:
I don't know if we want `indexed` explicitly called out in text here (also if we do you can maybe re-use the `MepTag` component)?

Reply:
It's internal only for our team